google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -3420,23 +3407,37 @@ public void testUploadWithEncryption() throws Exception {
   }
 
   @Test
-  public void testGetNotification() {
-    Notification actualNotification =
-        storage.getNotification(BUCKET, notification.getGeneratedId());
-    assertEquals(CUSTOM_ATTRIBUTES, actualNotification.getCustomAttributes());
-    assertEquals(PAYLOAD_FORMAT.name(), actualNotification.getPayloadFormat().name());
-    assertTrue(actualNotification.getTopic().contains(TOPIC));
-  }
-
-  @Test
-  public void testListNotification() {
-    List<Notification> notifications = storage.listNotifications(BUCKET);
-    for (Notification actualNotification : notifications) {
-      if (actualNotification.getGeneratedId().equals(notification.getGeneratedId())) {
-        assertEquals(CUSTOM_ATTRIBUTES, actualNotification.getCustomAttributes());
-        assertEquals(PAYLOAD_FORMAT.name(), actualNotification.getPayloadFormat().name());
-        assertTrue(actualNotification.getTopic().contains(TOPIC));
-      }
+  public void testNotification() throws InterruptedException, ExecutionException {
+    String bucketName = RemoteStorageHelper.generateBucketName();
+    Bucket bucket = storage.create(BucketInfo.newBuilder(bucketName).setLocation("us").build());
+    Notification bucketNotification =
+        Notification.newBuilder(TOPIC)
+            .setCustomAttributes(CUSTOM_ATTRIBUTES)
+            .setPayloadFormat(PAYLOAD_FORMAT)
+            .build();
+    try {
```

Comment:
you can check that notificationList is empty at this point.

Reply:
done